File: python3/1160.py
teste = int(input())

for caso in range(1, teste+1):
    pa, pb, ca, cb = input().split(" ")
    pa = int(pa)
    pb = int(pb)
    ca = float(ca)
    cb = float(cb)

    anos = 0
    while(pa <= pb and anos <= 100):
        pa += int(pa * (ca / 100))
        pb += int(pb * (cb / 100))
        anos += 1

    if(anos > 100):
        print("Mais de 1 seculo.")
    else:
        print("{} anos.".format(anos))

File: c++/1160.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main() {
    ///freopen ("entrada.in","r", stdin);
    int i, n, pa, pb, t;
    double ca, cb;
    t = 0;

    cin >> n;

    for(i = 1; i <= n; i++) {
        cin >> pa >> pb >> ca >> cb;
        while(pa <= pb and t <= 100) {
            pa += pa * (ca / 100.0);
            pb += pb * (cb / 100.0);
            t++;
        }
        if(t > 100) {
            cout << "Mais de 1 seculo." << endl;

        } else {
            cout << t << " anos." <<endl;
        }
        t = 0;
    }

    return 0;
}

